src/Symfony/Component/Intl/Tests/DateFormatter/AbstractIntlDateFormatterTest.php

```diff
@@ -355,13 +351,9 @@ public function testFormatWithConstructorTimezone()
 
     public function testFormatWithDateTimeZone()
     {
-        if (defined('HHVM_VERSION_ID')) {
-            $this->markTestSkipped('This test cannot work on HHVM. See https://github.com/facebook/hhvm/issues/5875 for the issue.');
-        }
-
-        $formatter = $this->getDateFormatter('en', IntlDateFormatter::MEDIUM, IntlDateFormatter::SHORT, new \DateTimeZone('GMT+03:00'), IntlDateFormatter::GREGORIAN, 'zzzz');
+        $formatter = $this->getDateFormatter('en', IntlDateFormatter::MEDIUM, IntlDateFormatter::SHORT, new \DateTimeZone('GMT'), IntlDateFormatter::GREGORIAN, 'zzzz');
```

Comment:
hmm, given the test failure, it looks like we have an issue with the support of ``GMT`` in the stub code